### ISSUE TITLE
stops working when page is zoomed out

### ISSUE THREAD

**Issue by aehmt** [philc/vimium#2212]
At the moment of this post, this behaviour only persist on learn.co . 

When page size is lower that 100%, Vimium stops responding to scrolling commands. 
Issue persists on different computers on learn.co .

**Comment by gdh1995** [comment]
Do you mean Vimium does not scroll for a page shorter than the screen? Or Vimium stops working wholly?

**Comment by aehmt** [comment]
No, on learn.co when page size is 100% scrolling using Vimium works fine but when you adjust the page size to 90% you lose scrolling using Vimium on that page. Page is not shorter than the screen. Scrolling using arrow keys or trackpad works but Vimium stops responding to scrolling commands.

**Comment by gdh1995** [comment]
This is indeed a bug, and it occurs if only a page is zoomed with a size of < 100%. Says we wants to scroll down an area:
* Vimium detects if an element is scrollable by increaing its scrollTop by 1 pixel
* If the zoom percentage is >= 100%, then this action succeeds.
* If the zoom percentage is < 100%, then this action fails without any error.

Although in my tests, for a zoom percentage of 90%, 2px is enough as a scroll step, it seems too hard to exactly know if the page is zoomed.
